index.html

```diff
@@ -189,16 +199,41 @@ <h2>Languages</h2>
                     </div>
                   </div>
                   <div class="Frame flex-row">
-                    <p>Frameworks</p>
+                    <h1 class="framework">Frameworks</h1>
                     <i class="fa fa-angle-right fa-2x" aria-hidden="true"></i>
                   </div>
                   <div class="skill flex-row">
-                    <p>Skills</p>
+                    <h4>Skills</h4>
                     <i class="fa fa-angle-right fa-2x" aria-hidden="true"></i>
                   </div>
                 </div>
               </section>
         </section>
+
+        <section>
+          <div class="contact-form">
+            <div class="contact-writeup">
+              <img src="Ellipse 3.svg" alt="" class="back-img">
+              <img src="Rectangle 4.svg" alt="" class="back-rec4">
+              <img src="Rectangle 7.svg" alt="" class="back-rec">
+              <h1>Contact me</h1>
+              <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding.
+                 I'd love to help with it</p>
+            </div>
+
+            <form method="post" action="https://formspree.io/f/xgebqelo">
+              <div class="contact-form-div">
+                <input type="text" placeholder="enter name" class="contact-input" maxlength="30" required>
+                <input type="text" placeholder="enter email" class="contact-input" required>
+                <textarea name="" id="" class="text-area" cols="30" rows="10" class="contact-text" placeholder="write your message here"
+                name="message" maxlength="500" required
+                ></textarea>
+                <img src="Ellipse 5.svg" alt="" class="back-circle">
+                <button>Get in touch</button>
+              </div>
+            </form>
```

Comment:
- Good job implementing the changes requested by your last reviewer [here](https://github.com/Developer-Kingz/my-portforlio-project/pull/4#discussion_r1188857398). However, now the form is not being sent to the Formspree.

![Screenshot from 2023-05-11 00-01-13](https://github.com/Developer-Kingz/my-portforlio-project/assets/97834160/dc9d5cb7-7ad4-43a4-80d7-b5dffadfbf04)

- Kindly ensure that you have included the "name" attribute. The "name" attribute is necessary for identifying the form fields when the data is sent to the server. You can add it like this `<input type="text" name="name" required>`.

- [OPTIONAL] Make sure to change the email field type to email, instead of text. It should be like this `<input type="email">`